core-web/libs/ui/src/lib/components/dot-content-thumbnail/dot-content-thumbnail.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="thumbnail" [ngSwitch]="thumbnailType">
+    <ng-container *ngSwitchCase="CONTENT_THUMBNAIL_TYPE.image">
+        <img
+            [src]="src"
+            [title]="name"
+            [alt]="name"
+            (error)="handleError()"
+            data-testId="thumbail-image" />
+    </ng-container>
+
+    <ng-container *ngSwitchCase="CONTENT_THUMBNAIL_TYPE.video">
+        <video (error)="handleError()" controls data-testId="thumbail-video">
+            <source [src]="src" />
+        </video>
+    </ng-container>
+
+    <ng-container *ngSwitchDefault>
+        <i
+            *ngSwitchDefault
+            [ngClass]="['pi', thumbnailIcon]"
+            [ngStyle]="{ 'font-size': iconSize, color: 'var(--color-palette-secondary-400)' }"
+            data-testId="thumbail-icon"></i>
+    </ng-container>
```

Comment:
The `ng-containers` are not needed in each case... you can put it directly in the element.

Reply:
Done!